lms/djangoapps/courseware/access_response.py

```diff
@@ -0,0 +1,85 @@
+"""
+This file contains all the classes used by has_access for error handling
+"""
+
+
+class AccessResponse(object):
+    """Class that represents a response from a has_access permission check."""
```

Comment:
I believe our coding conventions suggest not using one-line docstrings, even for one-line docs.